crates/matrix-sdk/src/oidc.rs

```diff
@@ -781,6 +790,15 @@ pub(crate) struct OidcData {
     pub(crate) latest_id_token: RwLock<Option<IdToken<'static>>>,
 }
 
+/// The data needed to perform authorization using OpenID Connect.
+#[derive(Debug)]
```

Comment:
```suggestion
#[derive(Debug, Clone)]
```
There's no harm in allowing to clone the struct and it can be useful for users.

Reply:
Thanks 👍. Intuitively when to add the various `derives` is still something I still haven't got into my head yet.